examples/notebooks/submodel_cracking_DFN_or_SPM.ipynb

```diff
@@ -0,0 +1,263 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#model = pybamm.lithium_ion.SPMe(\n",
+    "model = pybamm.lithium_ion.DFN(\n",
+    "    build=False,options = {\n",
+    "        \"particle\": \"Fickian diffusion\",  \n",
+    "        \"thermal\": \"lumped\", \n",
+    "        \"particle cracking\": \"both\", # other options are \"positive\", \"negative\" or \"none\"\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This collects all of the submodels which make up the DFN, but doesn't build the model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "param = model.default_parameter_values\n",
+    "\n",
+    "import pandas as pd\n",
+    "mechanics = pd.read_csv(\"pybamm/input/parameters/lithium-ion/mechanicals/lico2_graphite_Ai2020/parameters.csv\", \n",
+    "                        index_col=0, comment=\"#\", skip_blank_lines=True, header=None)[1][1:].dropna().astype(float).to_dict()\n",
+    "param.update(mechanics, check_already_exists=False)\n",
+    "## It can update the speed of crack propagation using the commands below:\n",
+    "# params.update({\"Negative electrode Number of cracks per unit area of the particle [m-2]\": 3.18e15/100})\n",
+    "# param.update({\"Negative electrode Cracking rate\":3.9e-20*1000})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can get the default parameters for the model and update them with the parameters required by the cracking model. Eventually, we would like these to be added to their won chemistry (you might need to adjust the path to the parameters file to your system).\n",
+    "Now the model can be processed and solved in the usual way, and we still have access to model defaults such as the default geometry and default spatial methods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "a8781709100142f5a754426317257ef9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='t', max=2.0868031269501226, step=0.02086803126950122…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "experiment = pybamm.Experiment([\n",
+    "    \"Discharge at 1C until 3.2 V\",\n",
+    "    \"Charge at 1C until 4 V\",\n",
+    "])\n",
+    "sim = pybamm.Simulation(\n",
+    "    model,\n",
+    "    experiment=experiment,\n",
+    "    parameter_values=param,\n",
+    "    solver=pybamm.CasadiSolver(dt_max=0.001),\n",
```

Comment:
this is a very conservative value of `dt_max`. 600 works fine for me

Reply:
will have a try and thanks